libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFJsonUtils.h

```diff
@@ -67,7 +67,16 @@
 
 
 /**
- Pull a value from the json-derived object by path ("." delimited)
+ * Pull a value from the json-derived object by path ("." delimited)
+ *
+ * Examples (in pseudo code):
+ *
+ * json = {"a": {"b": [{"c":"xx"}, {"c":"xy"}, {"d": [{"e":1}, {"e":2}]}, {"d": [{"e":3}, {"e":4}]}] }}
+ * projectIntoJson(jsonObj, "a") = {"b": [{"c":"xx"}, {"c":"xy"}, {"d": [{"e":1}, {"e":2}]}, {"d": [{"e":3}, {"e":4}]} ]}
+ * projectIntoJson(json, "a/b") = [{c:"xx"}, {c:"xy"}, {"d": [{"e":1}, {"e":2}]}, {"d": [{"e":3}, {"e":4}]}]
```

Comment:
I know this is pseudo-code, but it would still probably be helpful to use our delimiter in the example, i.e. `projectIntoJson(json, "a.b")`.